SpokestackTests/TFLiteWakewordRecognizerTest.swift

```diff
@@ -0,0 +1,234 @@
+//
+//  TFLiteWakewordRecognizerTest.swift
+//  SpokestackTests
+//
+//  Created by Noel Weichbrodt on 5/22/20.
+//  Copyright © 2020 Spokestack, Inc. All rights reserved.
+//
+
+import Foundation
+import CryptoKit
+@testable import TensorFlowLite
+@testable import Spokestack
+import XCTest
+
+class TFLiteWakewordRecognizerTest: XCTestCase {
+    
+    func hexStringToData(hexString: String) -> Data? {
+        let len = hexString.count / 2
+        var data = Data(capacity: len)
+        for i in 0..<len {
+            let j = hexString.index(hexString.startIndex, offsetBy: i*2)
+            let k = hexString.index(j, offsetBy: 2)
+            let bytes = hexString[j..<k]
+            if var num = UInt8(bytes, radix: 16) {
+                data.append(&num, count: 1)
+            } else {
+                return nil
+            }
+        }
+        return data
+    }
+    
+    override func setUp() {
+        //let filterHexString = filter.map { String(format: "%02hhx", $0) }.joined()
```

Comment:
dead code

Reply:
would like to revisit the idea, so I'm keeping the exploration in a test.